WooCommerce/WooCommerceScreenshots/Screens/Login/LoginPasswordScreen.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let passwordTextField = "Password"
+    static let loginButton = "Password Next Button"
+    static let errorLabel = "pswdErrorLabel"
+}
+
+class LoginPasswordScreen: BaseScreen {
+    let passwordTextField: XCUIElement
+    let loginButton: XCUIElement
+
+    init() {
+        passwordTextField = XCUIApplication().secureTextFields[ElementStringIDs.passwordTextField]
+        loginButton = XCUIApplication().buttons[ElementStringIDs.loginButton]
+        super.init(element: passwordTextField)
+
+        XCTAssert(passwordTextField.waitForExistence(timeout: 3))
+    }
+
+    func proceedWith(password: String) -> LoginEpilogueScreen {
+        _ = tryProceed(password: password)
+
+        return LoginEpilogueScreen()
+    }
+
+    func tryProceed(password: String) -> LoginPasswordScreen {
+
+        passwordTextField.tap()
+        passwordTextField.typeText(password)
+
+        XCTAssert(loginButton.waitForExistence(timeout: 3))
+        XCTAssert(passwordTextField.waitForHittability(timeout: 3))
```

Comment:
Should this be `loginButton` instead of `passwordTextField`? (I really like the improvements you've made here and elsewhere with the assertions.)

Reply:
Ah good catch – fixed, thanks!!